HopsanGUI/GUIObjects/GUIContainerObject.cpp

```diff
@@ -1486,30 +1486,45 @@ void ContainerObject::copySelected(CopyStack *xmlStack)
     appendCoordinateTag(*copyRoot, center.x(), center.y());
 
     // Copy components
+    const auto thisSystemsParameterNames = getParameterNames();
```

Comment:
I'm not sure if I like this variable name. Can't it just be called "systemParameterNames"?

Reply:
When I tried to understand my old dusty code to fix this the name confused me. Since this code deals with figuring out what system parameters are used. This new name is more clear and may reduce confusion the next time someone tries to understand that code. But yes the name is silly